src/provider/application.js

```diff
@@ -3,16 +3,17 @@ import { fixedTimeCompare } from '../lib/string';
 import { EventEmitter } from 'events';
 import URI from '../lib/uri';
 import logger from '../lib/logger';
-import { calculateHeight, calculateWidth } from '../lib/dimension';
+import { getOffsetToBody, calculateHeight, calculateWidth } from '../lib/dimension';
 import MutationObserver from 'mutation-observer';
 
 
 /** Application class which represents an embedded application. */
 class Application extends EventEmitter {
-  init({ acls = [], secret = null, onReady = null }) {
+  init({ acls = [], secret = null, onReady = null, targetNodes = [] }) {
```

Comment:
targetNodes is an array of css-selectors.

Reply:
should probably add function doc to this method as its starting to get big.  Also, I think targetNodes should just be passed into resizeConfig and probably called "targetSelector" or something more descriptive of what it is. Nodes sounds like its DOM nodes and not a query selector.

Reply:
I guess if we do that then the provider can't set their own target node configuration. FWIW, I'm wondering if the  resizeConfiguration should just go away and be flag of dynamic or fixed... with dynamic always being bodyOffset + optional target nodes (which could be configured in the provider or consumer)